<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editarCliente</class>
 <widget class="QDialog" name="editarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #cffdff;
}

QPushButton{
	background-color: #afdcde;
	border-radius: 14px;
	border: 2px solid #3d5557;
	color: #3d5557;
	font-size: 16px;
}

QPushButton::hover{
	background-color: #b9faf4;
}

QLabel{
	color: #3d5557;
	font-weight: bold;
}

QLineEdit {
	border-radius: 4px;
	border: 1px solid #3d5557
}
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>401</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_nome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_username"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_senha">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CPF:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_cpf">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Endere√ßo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_endereco"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cadastrar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
